saleor/order/utils.py

```diff
@@ -135,6 +135,16 @@ def update_order_prices(order, discounts):
             )
 
             price = manager.calculate_order_line_unit(order, line, variant, product)
+            if price is not None:
```

Comment:
Honestly, I don't understand the concept of this logic. Can you explain it?
- price always will be  different than None. When we don't have tax plugin enabled we will use price for calculations.py.
- You should not operate on .amount values. you should just call unit_price * quantity.  With that case, .quantitze is not needed. 
- I would recommend to update to the newest master. It contains changes in this method, but again IMO this part of code is not needed.